### DISCUSSION TITLE
Suggestion:   Add option to automatically place translation on clipboard

### DISCUSSION THREAD

**Discussion by Andy2No** [sienori/simple-translate#469] [Ideas]
I'm currently looking through some source code with comments in Chinese, which has been saved as a web page so that I can use Simple Translate in Firefox (for Windows).  I can get Simple Translate to translate a comment, but I then have to select the text by dragging before I can copy it to the clipboard.  That gets tiring quickly.

Using Ctrl-A doesn't work because it selects the translation plus the content of the entire page.

Please add an option to automatically place the translation on the clipboard, so it's ready to be pasted into another document containing the translation, one phrase or line at a time.

**Comment by mrshafaq** [comment]
i was thinking for same to have have copied upon selection .

**Comment by Magalaan** [comment]
I came here with the same suggestion to have an option to copy translation to keyboard automatically. I too use translated internet content to make descriptions of all kinds of stuff that I paste info files. I would be big time saver. 

By the way: great extension!
